lib/capdrupal.rb

```diff
@@ -0,0 +1,310 @@
+
+            # If we found less file than the keep number finish the task here
+            next "" unless backups.count > keep_backups.to_i
+
+            # Calculate number of file to delete.
+            to_delete = backups.count - keep_backups.to_i
+
+            # Loop throught file to delete (oldest files on top).
+            backups[0, to_delete].map do |backup|
+              execute :rm, "#{backup_path}/#{backup}"
+            end
+          end
+        end
+      end
+    end
+  end
+
+  namespace :cache do
+    desc 'Clear all caches'
+    task :clear do
+      on roles(:app) do
+        within release_path.join(fetch(:app_path)) do
+          execute :drush, 'cr'
+        end
+      end
+    end
+  end
+
+  namespace :module do
+    desc 'Enable module(s)'
+    task :enable do
+      on roles(:app) do
+        within release_path.join(fetch(:app_path)) do
+          for mod in fetch(:enable_modules)
+            execute :drush, "en '#{mod}' -y"
+          end
+        end
+      end
+    end
+
+    desc 'Disable module(s)'
+    task :disable do
+      on roles(:app) do
+        within release_path.join(fetch(:app_path)) do
+          for mod in fetch(:disable_modules)
+            execute :drush, "pmu '#{mod}' -y"
+          end
+        end
+      end
+    end
+  end
+
+  namespace :maintenance do
+    desc "Set maintenance mode"
+    task :on do
+      on roles(:app) do
+        within release_path.join(fetch(:app_path)) do
+          execute :drush, "state:set system.maintenance_mode 1 -y"
+          execute :drush, 'cr'
+        end
+      end
+    end
+
+    desc "Remove maintenance mode"
+    task :off do
+      on roles(:app) do
+        within release_path.join(fetch(:app_path)) do
+          execute :drush, "state:set system.maintenance_mode 0 -y"
+          execute :drush, 'cr'
+        end
+      end
+    end
+  end
+
+  desc 'Update database with migrations scripts'
+  task :updatedb do
+    on roles(:app) do
+      within release_path.join(fetch(:app_path)) do
+        execute :drush, 'updatedb -y'
+      end
+    end
+  end
+
+  namespace :updatedb do
+    desc 'Update database with migrations scripts'
+    task :silence do
+      on roles(:app) do
+        within release_path.join(fetch(:app_path)) do
+          execute :drush, 'updatedb -y', raise_on_non_zero_exit: false
+        end
+      end
+    end
+  end
+
+  desc 'Apply pending entity schema updates'
+  task :entup do
+    on roles(:app) do
+      within release_path.join(fetch(:app_path)) do
+        execute :drush, 'entup -y'
+      end
+    end
+  end
+
+  namespace :config do
+    desc 'Import configuration to active stage'
+    task :import do
+      on roles(:app) do
+        try = 0
+        full_config_path = release_path.join(fetch(:config_path))
+        within release_path.join(fetch(:app_path)) do
+          execute :drush, "config-import -y --source=#{full_config_path}"
```

Comment:
This exact line is a real pain in the A** because of that https://www.drupal.org/project/config_ignore/issues/2883859